pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -372,6 +370,23 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 	return false, nil
 }
 
+func nodeStatusDiffMessage(newCurrNodeState *operatorv1.NodeStatus, currNodeState *operatorv1.NodeStatus) string {
+	compareLog := fmt.Sprintf("Node %s at revision %d targets revision %d.", newCurrNodeState.NodeName, newCurrNodeState.CurrentRevision, newCurrNodeState.TargetRevision)
+	if currNodeState.LastFailedRevision != newCurrNodeState.LastFailedRevision {
+		compareLog += fmt.Sprintf(" The last failed revision changed from %d to %d.", currNodeState.LastFailedRevision, newCurrNodeState.LastFailedRevision)
+	} else {
+		compareLog += fmt.Sprintf(" The last failed revision remains %d.", newCurrNodeState.LastFailedRevision)
```

Comment:
looks mostly good now.

I think we should drop the message for unchanged field.

Reply:
cc @mfojtik ptal

Reply:
Done

Reply:
The no-change message is still here. Does it bring value?

Reply:
I think it's not fetching the changes. https://github.com/drichtarik/library-go/commit/57a279c8db5d296d9317b692aff22bf66090ab89